### DISCUSSION TITLE
BIOS files to be added in subfolder or directly in Folder?

### DISCUSSION THREAD

**Discussion by saurabh2804** [archtaurus/RetroPieBIOS#14] [Q&A] [answered]
The `bios_file` goes like this:
BIOS/`bios_file` or like this:
BIOS/Console_name/`bios_file`

And once added, will it be picked up by RetroPie automatically (after restart)?

**Comment by archtaurus** [comment, marked as answer]
I do BIOS/bios_file. But maybe both works. Or follow the RetroPie document instruction such as https://retropie.org.uk/docs/3do/ for example: 3do bios file should be put into /home/pi/RetroPie/BIOS.

In fact I did not do all the working test for all the bios. I just put all into /home/pi/RetroPie/BIOS and all "my games" works.
